### DISCUSSION TITLE
Scroll wheel in tile menu

### DISCUSSION THREAD

**Discussion by Lewis-Cooper101** [AmyE123/TactiKit#23] [Map Editor Feedback]
This is a very very minor quality of life piece of feedback, it does not impact the use of the map editor and still functions as intended. 

Basically at the moment when you use the scroll wheel to navigate in the tile menu, it is still zooming in/ out the camera in the scene and could be nice if they were not linked together. Again very minor, and is just something I noticed whilst using it.

**Comment by Vasilii098** [comment]
)))
